docs/streams/developer-guide.html

```diff
@@ -18,7 +18,7 @@
 <script><!--#include virtual="../js/templateData.js" --></script>
 
 <script id="content-template" type="text/x-handlebars-template">
-    <h1>Developer Guide</h1>
+    <h1>Developer Manual</h1>
```

Comment:
@derrickdoo thought `manual` is a better word for developers than `guide`. I'm fine either way.